semantic-python/src/Prelude.score

```diff
@@ -8,15 +8,10 @@
   // object's superclass is type itself
   object <- type "object" type #record{};
 
-  str <- type "str" object #record { };
-  str.__new__ = (\contents -> instance str contents #record{});
+  str <- type "str" object #record { __new__: \prim -> instance #unit prim #record{} };
```

Comment:
It turns out that we can’t yet assign to unbound fields within a record, so until we fix that I’ve backed out the parts of the prelude that assumed it would work.